test_conformance/compiler/test_compiler_defines_for_extensions.cpp

```diff
@@ -49,8 +49,10 @@ const char *known_extensions[] = {
     "cl_khr_subgroup_shuffle",
     "cl_khr_subgroup_shuffle_relative",
     "cl_khr_subgroup_clustered_reduce",
+    "cl_khr_subgroup_rotate"
```

Comment:
Nitpick: I believe this list follows the order of the [extension spec](https://registry.khronos.org/OpenCL/specs/3.0-unified/html/OpenCL_Ext.html). cl_khr_subgroup_rotate is in Section 48, so I would put it after cl_khr_integer_dot_product (in Section 42).

Reply:
that ordering makes sense, updated and left a comment above the array about this.

Reply:
Thanks, good to have a comment indeed.